### ISSUE TITLE
train on cityscapes but got bad result of mask

### ISSUE THREAD

**Issue by myBestLove** [matterport/Mask_RCNN#1453]
I change the cityscapes to coco annotations file  and load mask like coco.py.

I finetune heads  on coco pretrained model by 30 epochs,   but got bad result of mask.

And I changed the train rules,  the mask was also bad ,  even worse than coco pretrained model. 

anyone can help me to train a good model on cityscape?  THX!
